cmd/generate/config/spec.go

```diff
@@ -0,0 +1,142 @@
+package config
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/charmbracelet/bubbles/help"
+	"github.com/charmbracelet/bubbles/key"
+	"github.com/charmbracelet/bubbles/textinput"
+	tea "github.com/charmbracelet/bubbletea"
+)
+
+// Bubbletea for Interactive Mode
+
+type model struct {
+	textInput textinput.Model
+	help      help.Model
+	keymap    keymap
+
+	opts           *Options
+	attributionMap map[string][]string
+	uniqueEmails   []string
+	currentIndex   int
+}
+
+type keymap struct{}
+
+func (k keymap) ShortHelp() []key.Binding {
+	return []key.Binding{
+		key.NewBinding(key.WithKeys("ctrl+n"), key.WithHelp("ctrl+n", "next")),
+		key.NewBinding(key.WithKeys("ctrl+p"), key.WithHelp("ctrl+p", "prev")),
+		key.NewBinding(key.WithKeys("ctrl+i"), key.WithHelp("ctrl+i", "ignore email")),
+		key.NewBinding(key.WithKeys("esc"), key.WithHelp("esc", "quit")),
+		key.NewBinding(key.WithKeys("enter"), key.WithHelp("enter", "submit")),
+	}
+}
+
+func (k keymap) FullHelp() [][]key.Binding {
+	return [][]key.Binding{k.ShortHelp()}
+}
+
+func initialModel(opts *Options, uniqueEmails []string) model {
+	ti := textinput.New()
+	ti.Placeholder = "name"
```

Comment:
This should be username. When I went through interactive mode, even though I know it's supposed to be the GitHub username, I actually typed in people's actual full names.

![CleanShot 2024-09-10 at 13 58 14](https://github.com/user-attachments/assets/9d9986a9-b3e3-495c-bcdc-7edf895ae6e3)